src/bun.js/bindings/sqlite/JSSQLStatement.cpp

```diff
@@ -488,6 +491,60 @@ class JSSQLStatement : public JSC::JSDestructibleObject {
     void finishCreation(JSC::VM& vm);
 };
 
+static JSValue toJSAsBuffer(JSC::VM& vm, JSC::JSGlobalObject* globalObject, sqlite3_stmt* stmt, int i)
+{
+    auto scope = DECLARE_THROW_SCOPE(vm);
+
+    switch (sqlite3_column_type(stmt, i)) {
```

Comment:
can you double check this is the right one to use? the one that is the actual value and not the declaration type